Outputs/SSD/Graph/SSD_graph.json

```diff
@@ -776,13 +776,13 @@
         0.0,
         0.0,
         0.0,
-        7.0,
-        7.0,
-        7.0,
-        7.0,
-        7.0,
-        7.0,
-        7.0
+        107.0,
+        107.0,
+        107.0,
+        107.0,
+        107.0,
+        107.0,
+        107.0
```

Comment:
@milanoleonardo This doesn't look like correct. There is a single day with confirmed cases for Yei with 7. Are you aggregating at the ADMIN1 level rather than ADMIN2?

Reply:
Aaah sorry I had a typo, good catch